### PR TITLE
Update news.md

### PR DESCRIPTION
**Pull request by neoatomic**

### CODE REVIEW

File: news.md [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
-All issues and error reports must be opened at [GitHub](https://github.com/SiCKRAGETV/sickrage-issues/issues) and you **MUST** follow **ALL** [guidelines](https://github.com/SiCKRAGETV/sickrage-issues#submitting-a-bugissue-ticket)
-
-We can track better errors reports in GitHub. Not so much in SickRage forum
+####Please read the [Wiki/FAQ](https://github.com/SiCKRAGETV/sickrage-issues/wiki) before opening an issue####
+All issues and bugs reports must be opened at [GitHub](https://github.com/SiCKRAGETV/sickrage-issues/issues) (not the forum!) and you **MUST** follow **ALL** [guidelines](https://github.com/SiCKRAGETV/sickrage-issues#submitting-a-bugissue-ticket)  
 
 <br/>
 
+####2015-09-09####
+- As you might have noted the [TVRage](http://www.tvrage.com) website pulled the plug/shut down. And that is causing some issues we needed to address. We pushed a fix that automatically converts the TVRage ID to a TheTVDB ID. So that new data is pulled from theTVDB.com instead. That works for most users but sadly not all. So be aware that when u used TVRage shows, you might experience some issues. Be assured that we are working hard to get this all sorted out.
```
Comment (duramato) [inline news.md]: You typed "u" instead of you , in ```that when u used TVRage```

**Comment by miigotu** [comment]
lol, PR to fix it. sickrage-news is open for PR from anyone.

@duramato we also need some changelog action xD

**Comment by duramato** [comment]
@miigotu i just made a pr for the typo
sure will dive in to making the changelog in a few

File: news.md (hunk 1 above)
Comment (neoatomic) [inline news.md]: Happens when you write 3 languages daily. :( 

Also see i have done only 1 space behind `get this all sorted out. ` So there is no empty line below the message. :(

**Comment by miigotu** [comment]
@duramato there are 3 releases. The last two are really small.

**Pull request merged**